### PR TITLE
fix(bleed): fix pack flow valves not closing when switching to ignition

### PR DESCRIPTION
**Pull request by BlueberryKing**
Fixes #[issue_no]

## Summary of Changes

There was an issue in the pack flow valve control logic that had it remain until an engine master switch was turned on, instead of closing the valves when the engine mode selector is switched to IGN. This PR should fix this behavior.

## References

image

## Additional context

Discord username (if different from GitHub): BlueberryKing#6641

## Testing instructions

- Make sure you can still start an engine.
- Make sure that you cannot hear the packs anymore when you turn the engine mode selector to ignition.
- Make sure the packs come back after engine start.

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: src/systems/systems/src/air_conditioning/acs_controller.rs [hunk 1]
```diff
@@ -668,14 +668,16 @@ impl<const ZONES: usize> PackFlowController<ZONES> {
...
                 || !pneumatic.engine_crossbleed_is_on())
+            && (pneumatic.engine_mode_selector() != EngineModeSelector::Ignition)
             && !engine_fire_push_buttons.is_released(1)
             && !pressurization_overhead.ditching_is_on();
-        // && ! pack 1 overheat
-        // Flow Control Valve 2
-        self.fcv_2_open_allowed = acs_overhead.pack_pushbuttons_state()[1]
+            // && ! pack 1 overheat
+            // Flow Control Valve 2
+            self.fcv_2_open_allowed = acs_overhead.pack_pushbuttons_state()[1]
```
Comment (tracernz) [inline src/systems/systems/src/air_conditioning/acs_controller.rs:676]:
```suggestion
        // && ! pack 1 overheat
        // Flow Control Valve 2
        self.fcv_2_open_allowed = acs_overhead.pack_pushbuttons_state()[1]
```
Indentation was correct before?

**Review by tracernz — APPROVED** [review]
LGTM, but maybe a test would be a good idea?

**Comment by BlueberryKing** [comment]
> LGTM, but maybe a test would be a good idea?

Good point, will get on with that!

**Comment by AquazDev** [comment]
Quality Assurance Tester/Trainee Report

Discord: LunakisLeaks#7744
Object of testing: #6805
Tier of Testing: 1
Date: 2/3/2022

Negatives:       
The Cameras is glitching

Testing Results: 
Passed

**Comment by AquazDev** [comment]
Quality Assurance Tester/Trainee Report

Discord: LunakisLeaks#7744
Object of testing: #6805
Tier of Testing: 1
Date: 2/3/2022

Testing Process: 
Engines Start Up - OK 
Cannot Hear The Packs Anymore - OK
Packs Come Back After Engine Start - OK

Negatives:
None

Testing Results: 
Passed

Conclusions: 
LGTM

Media:
None

**Comment by NUTTTTTTTT** [comment]
Quality Assurance Tester/Trainee Report

Discord: █▀█ █▄█ ▀█▀#2123
Object of testing: #PR 6805
Tier of Testing: 1
Date: 02/03/2022

Testing Process: 
- Started Cold and Dark, switched batts on, APU started. 
- Checked for audible pack sound - OK.
- Verified pack sound goes when switching engine master switch to IGNITION - OK.
- Engines 1 started, pack sound still not audible.
- Engine 2 started, pack sound came back after successful engine start.

- Shut down both engines, checked pack sound still cuts out when master switch is set to ignition - OK.
- Repeated start up sequence with engine 2 then 1, checked for same behaviour - OK.
- Checked Pack 1+2 buttons still work as expected - OK.

Negatives:   N/A

Testing Results: Passed

Conclusions: Works as intended, LGTM!

**Pull request merged**